includes/definitions/ciscosb.yaml

```diff
@@ -34,6 +34,7 @@ discovery_modules:
 discovery:
     - sysObjectID:
             - .1.3.6.1.4.1.9.6.1.23.3.13
+            - .1.3.6.1.4.1.9.6.1.72
```

Comment:
You should probably add yet another snmprec file for ciscosb.  I wonder if a regex could be used in here.

Reply:
Added smprec file for sge variant. I don't know if it's posible to put regex in yaml.